src/components/pages/landingpage/career.tsx

```diff
@@ -5,11 +5,23 @@ import { HomePageHeaderBlock } from './support';
 import { SyPersonioJob } from '../../../types';
 import { Button } from '../../ui/buttons/button';
 import { LandingPageTeaserGrid } from './landing-page-teaser-grid';
+import styled from 'styled-components';
+import { up } from '../../support/breakpoint';
 
 interface CareerProps {
   positions: SyPersonioJob[];
 }
 
+const CareerTeaser = styled(Teaser)`
```

Comment:
Oh I wonder why this code is duplicated. Not worth a flag on the teaser?

Reply:
A flag is much more elegant, I added it 👍